projects/fsastorefrontlib/src/cms-components/my-account/address-info/address-form/address-form.component.html

```diff
@@ -0,0 +1,158 @@
+        formControlName="lastName"
+        readonly
+      />
+      <cx-form-errors [control]="addressForm.get('lastName')"></cx-form-errors>
+    </label>
+  </div>
+  <div class="form-group">
+    <label>
+      <span class="label-content required">{{
+        'addressForm.streetName' | cxTranslate
+      }}</span>
+      <input
+        type="text"
+        class="form-control"
+        [placeholder]="'addressForm.streetAddress' | cxTranslate"
+        formControlName="line1"
+      />
+      <cx-form-errors [control]="addressForm.get('line1')"></cx-form-errors>
+    </label>
+  </div>
+  <div class="form-group">
+    <label>
+      <span class="label-content">{{
+        'addressForm.streetNumber' | cxTranslate
+      }}</span>
+      <input
+        type="text"
+        class="form-control"
+        [placeholder]="'addressForm.aptSuite' | cxTranslate"
+        formControlName="line2"
+      />
+    </label>
+  </div>
+  <div class="row">
+    <div class="form-group col-md-6">
+      <label>
+        <span class="label-content required">{{
+          'addressForm.city.label' | cxTranslate
+        }}</span>
+        <input
+          type="text"
+          class="form-control"
+          [placeholder]="'addressForm.city.placeholder' | cxTranslate"
+          formControlName="town"
+        />
+        <cx-form-errors [control]="addressForm.get('town')"></cx-form-errors>
+      </label>
+    </div>
+    <div class="form-group col-md-6">
+      <label>
+        <span class="label-content required">{{
+          'addressForm.zipCode.label' | cxTranslate
+        }}</span>
+        <input
+          type="text"
+          class="form-control"
+          [placeholder]="'addressForm.zipCode.placeholder' | cxTranslate"
+          formControlName="postalCode"
+        />
+        <cx-form-errors
+          [control]="addressForm.get('postalCode')"
+        ></cx-form-errors>
+      </label>
+    </div>
+    <ng-container *ngIf="regions$ | async as regions" formGroupName="region">
+      <ng-container *ngIf="regions.length !== 0">
+        <div class="form-group col-md-6">
+          <label aria-required="true">
+            <span class="label-content required">{{
+              'addressForm.state' | cxTranslate
+            }}</span>
+            <ng-select
+              class="region-select"
+              formControlName="isocode"
+              [searchable]="true"
+              [clearable]="false"
+              [items]="regions"
+              [bindLabel]="regions[0].name ? 'name' : 'isocode'"
+              [bindValue]="regions[0].name ? 'isocode' : 'region'"
+              [placeholder]="'addressForm.selectOne' | cxTranslate"
+            >
+            </ng-select>
+            <cx-form-errors
+              [control]="addressForm.get('region.isocode')"
+            ></cx-form-errors>
+          </label>
+        </div>
+      </ng-container>
+    </ng-container>
+  </div>
+  <div class="cx-checkout-btns form-group row">
```

Comment:
`cx-checkout-btns` class is out of place here, and if I'm not mistaken has no styling/positioning impact at all, so it can be removed.